langlib/lang.error/src/main/ballerina/error.bal

```diff
@@ -18,7 +18,7 @@ import ballerina/jballerina.java;
 
 # Type for value that can be cloned.
 # This is the same as in lang.value, but is copied here to avoid a dependency.
-type Cloneable readonly|xml|Cloneable[]|map<Cloneable>|table<map<Cloneable>>;
+public type Cloneable readonly|xml|Cloneable[]|map<Cloneable>|table<map<Cloneable>>;
```

Comment:
Shall we also send a PR to the spec repo also?

Reply:
https://github.com/ballerina-platform/ballerina-spec/pull/1063